views/database/edit.erb

```diff
@@ -0,0 +1,91 @@
+<div class="container">
+  <form action='/dbs' method='post'>
+    <input type="hidden" name="_method" value="patch">
+    <input type="hidden" name="id" value="<%= @db.id %>">
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Database Name</th>
+          <th>Old</th>
+          <th>New</th>
+        </tr>
+      </thead>
+      
+      <tbody>
+        <tr>
+          <td>Name</td>
+          <td>
+            <% if @db %>
+              <%= @db.name %>
+            <% else %>
+              N/A
+            <% end %>
+          </td>
+          <td>
+            <input type="text" name="name" placeholder="New name">
+          </td>
+        </tr>
+        <tr>
+          <td>Project Id</td>
+          <td>
+            <% if @db %>
+              <%= @db.project_id %>
```

Comment:
Whenever we display the project in a form or page, we should probably display its name, not its id, which won't mean much to people